applications/RomApplication/python_scripts/rom_nn_trainer.py

```diff
@@ -57,13 +53,15 @@ def _CheckNumberOfModes(self,n_inf,n_sup,n_max):
             err_msg = f'Specified number of superior modes ({n_sup}) is higher than the available ones from the Phi matrix ({n_max}).'
             raise Exception(err_msg)
 
-    def _GetTrainingData(self, n_inf, n_sup, database_settings):
-        
-        S_train = np.load(database_settings['training_set'].GetString())
-        S_val = np.load(database_settings['validation_set'].GetString())
+    def _GetTrainingData(self, n_inf, n_sup):
 
-        phi = np.load(database_settings['phi_matrix'].GetString())
-        sigma_vec = np.load(database_settings['sigma_vector'].GetString())/np.sqrt(S_train.shape[1])
+        S_train = self.data_base.get_snapshots_matrix_from_database(self.mu_train, table_name=f'FOM_Fit')
+        S_val = self.data_base.get_snapshots_matrix_from_database(self.mu_validation, table_name=f'FOM_Fit')
```

Comment:
Maybe it would be good to throw an exception if not all the snapshots for the datasets were found, although it's true that in that case the check_in_database for the RightBasis will fail too.